src/tests/ftest/util/ior_test_base.py

```diff
@@ -227,6 +228,44 @@ def check_subprocess_status(self, operation="write"):
                 self.job_manager.process, self.ior_cmd):
             self.fail("Exiting Test: Subprocess not running")
 
+    def check_pool_free_space(self, expected_scm=None, expected_nvme=None,
```

Comment:
Optional - Since the only `self` reference this function really needs is `self.fail`, I'd recommend moving this function to a base class. For example, `TestWithServers` (which this class inherits) https://github.com/daos-stack/daos/blob/38ec717e8380758dce844ae905719c225f8884d6/src/tests/ftest/util/apricot/apricot/test.py#L481
Then, the function is available to other test bases as well.